src/unitxt/test_utils/card.py

```diff
@@ -26,44 +27,38 @@ def test_metrics_exist(card):
 
 def test_loading_from_catalog(card):
     with tempfile.TemporaryDirectory() as tmp_dir:
-        add_to_catalog(card, TEMP_NAME, overwrite=True, catalog_path=tmp_dir)
+        add_to_catalog(card, TEMP_NAME, overwrite=True, catalog_path=tmp_dir, verbose=False)
         register_local_catalog(tmp_dir)
         card_, _ = fetch_artifact(TEMP_NAME)
         assert json.dumps(card_.to_dict(), sort_keys=True) == json.dumps(
             card.to_dict(), sort_keys=True
         ), "Card loaded is not equal to card stored"
 
 
-def load_examples_from_common_recipe(card, tested_split):
-    if card.templates:
-        num_templates = len(card.templates)
+def load_examples_from_standard_recipe(card, tested_split, template_card_index):
+    print("=" * 80)
+    print(f"Using template card index: {template_card_index}")
 
-        try:  # named templates (dict)
-            template_item = next(iter(card.templates.keys()))
-        except AttributeError:  # template list
-            template_item = 0
-    else:
-        num_templates = 0
-        template_item = None
     num_instructions = len(card.instructions) if card.instructions else 0
-    recipe = CommonRecipe(
+    recipe = StandardRecipe(
         card=card,
         demos_pool_size=100,
         demos_taken_from=tested_split,
         num_demos=3,
-        template_item=template_item,
-        instruction_item=0 if num_instructions else None,
+        template_card_index=template_card_index,
+        instruction_card_index=0 if num_instructions else None,
```

Comment:
this is never None according to line 48

Reply:
num_instructions is never None, but 
  
  "if num_instructions " does not check if the (int) variable is None, it checks if it's non zero:
  
  https://stackoverflow.com/questions/73241173/what-is-the-meaning-of-the-if-0-in-python